refactron/cicd/quality_gates.py

```
@@ -0,0 +1,231 @@
+        Raises:
+            ValueError: If JSON is invalid
+            FileNotFoundError: If file doesn't exist
+        """
+        if not json_path.exists():
+            raise FileNotFoundError(f"JSON output file not found: {json_path}")
+
+        try:
+            with open(json_path, "r", encoding="utf-8") as f:
+                return json.load(f)
+        except json.JSONDecodeError as e:
+            raise ValueError(f"Invalid JSON in output file: {e}") from e
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)